src/Table/Tr.php

```
@@ -26,6 +27,52 @@
+     * Usage example:
+     * ```php
+     * $tr = \UIAwesome\Html\Table\Tr::tag()->cells('Jane', '30', 'NYC');
+     * ```
+     *
+     * @param string|Stringable ...$values Content for each `<td>` cell.
+     *
+     * @return static New instance with the appended data cells.
+     */
+    public function cells(string|Stringable ...$values): static
+    {
+        $tr = $this;
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $tr. Configured minimum length is 3. (undefined)

(ShortVariable)